docs/api-reference/stats.rst

```diff
@@ -0,0 +1,50 @@
+STATS API
```

Comment:
```suggestion
Stats API
```

Reply:
JSON API uses all caps so I did. But will change.

Reply:
Right, because JSON is an acronym -- "Stats" is not (see also the "Legacy API")